lib/navigations/home_tab.dart

```diff
@@ -65,8 +65,8 @@ class _HomeTabState extends State<HomeTab> {
         type: BottomNavigationBarType.shifting,
         currentIndex: _index,
         onTap: (int index) => setState(() => this._index = index),
-        selectedItemColor: Theme.of(context).textTheme.headline1!.color,
-        unselectedItemColor: Theme.of(context).textTheme.headline1!.color,
+        selectedItemColor: Theme.of(context).textTheme.displayLarge!.color,
+        unselectedItemColor: Theme.of(context).textTheme.displayLarge!.color,
```

Comment:
Could you kindly put the reference to this naming changes?

Reply:
I added an explanatory screenshot for the changed theme

Reply:
Thanks it looks great!